tensorflow/java/src/main/java/org/tensorflow/Output.java

```diff
@@ -57,6 +60,30 @@ public Output asOutput() {
     return this;
   }
 
+  @Override
+  public int hashCode() {
+    return operation.hashCode() + 37 * index + 17;
```

Comment:
Why this choice?
Actually, would it make sense to use [`Objects.hash(operation, index)`](https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#hash(java.lang.Object...)) instead?